### PR TITLE
Fix depratiation in Julia 1.0

### PR DESCRIPTION
**Pull request by amgad-naiem**

### CODE REVIEW

**Comment by anriseth** [comment]
Thanks for the PR. I was planning to wait until JuMP 0.19 was out but if someone wants to fix this sooner that's great  :)

I think this will break 0.6 support, so could you please update REQUIRE, Travis, and AppVeyor to reflect the move to 1.0?

File: REQUIRE [hunk 1]
```diff
@@ -1,6 +1,8 @@
-julia 0.6
+julia 0.7
 JuMP 0.17.1
 Ipopt
 Combinatorics
 RecipesBase
 LaTeXStrings
+MathProgBase
+LinearAlgebra
```
Comment (anriseth) [inline REQUIRE]: LinearAlgebra does not need to be included here

Comment (amgad-naiem) [reply REQUIRE]: yes. I just removed it and added Project.toml. Tests should work fine now

File: .travis.yml [hunk 2]
```diff
@@ -14,9 +14,9 @@ notifications:
...
   apt_packages:
     - gfortran
-script:
-  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  #- julia -e 'Pkg.add("Ipopt")'
-  - julia -e 'Pkg.clone(pwd()); Pkg.test("MultiJuMP", coverage=true)'
+# script:
+#   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+#   #- julia -e 'Pkg.add("Ipopt")'
+#   - julia -e 'using Pkg; Pkg.clone(pwd()); Pkg.test("MultiJuMP", coverage=true)'
```
Comment (anriseth) [inline .travis.yml]: You seem to have disabled the CI tests?

**Comment by anriseth** [comment]
We're getting closer;) Seems like an issue with the `using` statement in `runtests.jl`. 

Did you check whether the tests pass locally before pushing? That often speeds up the process as you don't have to wait for Travis.

**Comment by anriseth** [comment]
Thank you:))

**Comment by anriseth** [comment]
Release 0.2.0 should work with Julia 1.0 https://github.com/JuliaLang/METADATA.jl/pull/18732

**Pull request merged**